### ISSUE TITLE
File xcframework not found (4.8.0)

### ISSUE THREAD

**Issue by pavelannin** [c-villain/YandexMapsMobileLite#2]
Please add the assets file with the framework for version 4.8.0

**Error:**
`failed downloading 'https://github.com/c-villain/YandexMapsMobileLite/releases/download/4.8.0/YandexMapsMobile.xcframework.zip' which is required by binary target 'YandexMapsMobile': badResponseStatusCode(404)
`

**Comment by c-villain** [comment]
oh, sry)) some bug from GitHub during loading...

**Comment by c-villain** [comment]
@pavelannin  fixed in new release  4.8.0

These scripts will help you with reseting caches:

```bash
rm -rf ~/Library/Caches/org.swift.swiftpm
rm -rf ~/Library/org.swift.swiftpm
```

**Comment by pavelannin** [comment]
@c-villain 
Thanks for the quick solution
